packages/legacy/src/common/constants.ts

```diff
@@ -106,10 +106,10 @@ export enum StepType {
 
 export const dnsLabelNameSchema = yup
   .string()
-  .max(63)
-  .matches(/^(?=.{1,253}$)[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/, {
+  .max(253)
+  .matches(/^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/, {
```

Comment:
it was nice to keep the exact regexp as the new code, but this works too, for next time, please keep changes in the legacy code to minimum.

Reply:
Please note that without this fix:
1. The validation will be for 63 max len instead of 63
2.   a dedicated error message of: 'xxx name must be at most 253 characters' won't appear.

Reply:
> The validation will be for 63 max len instead of 253

63 id fine, this code is working since the begining, it hasn't bothered anyone until now, and will not in the future.

> a dedicated error message of: 'xxx name must be at most 253 characters' won't appear.

that is fine too, name is good enough, again this code worked from the first version of this application.

note:
when we replace this code, we will make it nicer, we have an issue to check validations and help texts in the provider forms #646, and we will have a similar task once we replace the mapping and plan forms.

on the other hand, we should not touch the legacy code, we don't want to maintain it, and every change we do, means we own the new code change and need to maintain it.